beetmoverscript/script.py

```diff
@@ -30,16 +31,30 @@
     context.task = get_task(context.config)  # e.g. $cfg['work_dir']/task.json
     # 2. validate the task
     validate_task_schema(context)
-    # 3. generate manifest
+    # 3 grab manifest props with all the useful data
+    context.properties = await get_props(context)
+    # 4. generate manifest
     manifest = generate_candidates_manifest(context)
-    # 4. for each artifact in manifest
+    # 5. for each artifact in manifest
     #   a. download artifact
     #   b. upload to candidates/dated location
     await move_beets(context, manifest)
-    # 5. copy to releases/latest location
+    # 6. copy to releases/latest location
```

Comment:
I think this logic has bitrotted and is part of move_beets. Feel free to remove the line.